src/components/Actions/Actions.vue

```diff
@@ -158,46 +159,85 @@ export default {
 			opened: this.open,
 			focusIndex: 0,
 			randomId: 'menu-' + GenRandomId(),
-			offsetX: 0
+			offsetX: 0,
+			// Making children reactive!
+			// By binding this here, vuejs will track the object content
+			// Needed for firstAction reactivity !!!
+			children: this.$children
 		}
 	},
+
 	computed: {
+		/**
+		 * Is there more than one action?
+		 */
+		hasMultipleActions() {
+			return this.actions.length > 1
+		},
+		/**
+		 * Is there any first action ?
+		 * And is it allowed as a standalone element ?
+		 */
 		isValidSingleAction() {
 			return this.actions.length === 1
+				&& this.firstActionElement !== null
```

Comment:
`this.firstActionElement` is always `undefined` (it's never defined anywhere after this PR). Should be `this.firstActionBinding` probably. I'll create a PR.

Reply:
PR in https://github.com/nextcloud/nextcloud-vue/pull/2878